spec/fixtures/application/1.0/application.json

```diff
@@ -244,9 +244,9 @@
       "dependants": [{"age": 1}, {"age":  5}],
       "manage_without_income": "other",
       "manage_other_details": "Another way they manage",
-      "applicant_self_assessment_tax_bill": "yes",
-      "applicant_self_assessment_tax_bill_amount": 55500,
-      "applicant_self_assessment_tax_bill_frequency": "fortnight",
+      "applicant_self_assessment_tax_bill": null,
+      "applicant_self_assessment_tax_bill_amount": null,
+      "applicant_self_assessment_tax_bill_frequency": null,
```

Comment:
These fixtures are for the unemployed flow, so I believe these values would be null. Correct me if i'm wrong!

Reply:
Update: agreed to leave for the time being as there would be more changes to be made in the fixture files. Instead, the devs need to have a wider conversation around having separate fixtures for different application types